lib/screens/SiteDetailScreen.dart

```diff
@@ -81,14 +81,7 @@ class _SiteDetailScreenState extends State<SiteDetailScreen> {
 
   @override
   Widget build(BuildContext context) {
-    final dnIcon =
-        Theme.of(context).brightness == Brightness.dark ? 'images/dn-logo-dark.svg' : 'images/dn-logo-light.svg';
-    final title = Row(children: [
-      site.managed
-          ? Padding(padding: EdgeInsets.only(right: 10), child: SvgPicture.asset(dnIcon, width: 12))
-          : Container(),
-      Expanded(child: Text(site.name, style: TextStyle(fontWeight: FontWeight.bold)))
-    ]);
+    final title = SiteTitle(site: widget.site);
```

Comment:
I guess `SiteTitle` can't be const since it accepts the dynamic prop `widget.site`...